onadata/apps/logger/models/xform.py

```diff
@@ -815,6 +816,9 @@ def save(self, *args, **kwargs):
         if 'skip_xls_read' in kwargs:
             del kwargs['skip_xls_read']
 
+        # update hash field
+        self.hash = self.get_hash()
```

Comment:
does this run on every save? can we run this only when we save and the new xml differs from the existing xml?